Added missing variables, modified inconsistent variables/description and broken links

acostak opened the pull request:
Added missing variables, modified inconsistent variables/description and broken links

Commit:
no message

Commit:
no message

lixun910 committed:
update to link new pages

lixun910 committed:
update url for css/js

lixun910 committed:
update geojson and links

lixun910 committed:
update configure

lixun910 committed:
update path

lixun910 committed:
update img

lixun910 committed:
update map

prevent mouse scroll on map

lixun910 committed:
category banner

lixun910 committed:
#24

when you click on any of the red banners, you get a 404 page not found
error
right now, the blue banner is too large relative to the map (see
attached): Could you make that smaller (same extent as text) and the
map bigger?
at https://geodacenter.github.io/data-and-lab/tags/, could you remove
the upload date since we only need the name?
let's disable comments at the bottom ("Comments: We were unable to load
Disqus. If you are a moderator please see our troubleshooting guide."

lixun910 committed:
update fix

lixun910 committed:
update tag fix

Commit:
test

Commit:
enforce UTF-8 encoding

Commit:
test2

Commit:
ohiolung

Commit:
updating AirBnB

lixun910 committed:
Update 2017-07-1-airbnb_Chicago 2015.md

lixun910 committed:
Update 2017-07-19-colomb_malaria.md

lixun910 committed:
Update 2017-07-19-colomb_malaria.md

lixun910 committed:
converting file to utf8

lixun910 committed:
reduce height of post title; update markers to simple dot for map points

lixun910 committed:
when mouse out the map, disable focus, so when mouse over again, it won't zoom

lixun910 committed:
update title size and the dot img url

lixun910 committed:
update title when shrink the width of the page

Commit:
Updated md files

lixun910 committed:
fix cosmetic erros

Commit:
Auto stash before merge of "gh-pages" and "origin/gh-pages"

Commit:
updated md files and geojsons

Commit:
more md and geojson updates

Commit:
md update

lixun910 committed:
Categories are case-sensitive, so remember all has to be lower-case

lixun910 committed:
Merge remote-tracking branch 'origin/gh-pages' into gh-pages

# Conflicts:
#	_posts/2017-05-29-baltimore_old.md

lixun910 committed:
update categories

lixun910 committed:
update categories

lixun910 committed:
update category link

lixun910 committed:
add economics category

lixun910 committed:
hide map when there is no data

Commit:
updated tags

Commit:
updated tags2

Commit:
ndvi md file fix

Commit:
tag update

Commit:
adding missing geojsolns + md fixes

Commit:
small fixes

Commit:
download data test

Commit:
download data2

lixun910 committed:
update

Commit:
download test

Commit:
test

Commit:
no message

Commit:
download test

Commit:
download links for all old sample data

lixun910 committed:
update tag order in index page

lixun910 committed:
remove testing code

Commit:
download links for new data

Commit:
scrime updated

Commit:
md fixes

Commit:
table test

Commit:
css

Commit:
index

Commit:
table

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
centering table

Commit:
no message

Commit:
updated table links

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

Commit:
no message

lixun910 committed:
update link with short length

Commit:
footer updates

lixun910 committed:
change index ui and test new tag page

lixun910 committed:
update filter by tag

lixun910 committed:
update table style

lixun910 committed:
update table style

lixun910 committed:
clean table data

Commit:
adding new data and style fixes

Commit:
fixing special characters

Commit:
more special character fixes

Commit:
no message

Commit:
fixed links

Commit:
no message

lixun910 committed:
fix 5,000 issue

Commit:
added tutorial page link

Commit:
no message

Commit:
no message

Commit:
updated substantive area tags

Commit:
no message

Commit:
tracking of individual sample data downloads

lixun910 committed:
update with non-unicode text

lixun910 committed:
fix: Income and Error in Phonix —> Phoenix

Commit:
Set up infrastructure for YAML dataset documentation

Commit:
Remove lines/fix formatting in README

Commit:
Include infrastructure for Google Dataset Schema

angela-li committed:
Merge pull request #27 from GeoDaCenter/dataset-schema

Include infrastructure for Google Dataset Schema

angela-li committed:
Add documentation for Community_pop.csv

angela-li committed:
Add link to Community_pop.csv on Amazon S3

angela-li committed:
Merge pull request #30 from GeoDaCenter/add-pop-homepage

Add Community pop data to data homepage

angela-li committed:
Add commpop data to homepage

angela-li committed:
Merge pull request #31 from GeoDaCenter/add-commpop

Add commpop data to homepage

lixun910 committed:
host download zip files

CHIM is about 700MB, and the raw matrices directories are dropped

lixun910 committed:
update  download links to github.com

lixun910 committed:
Update index.html

lixun910 committed:
Update index.html

lixun910 committed:
Update 2017-07-19-CHIM.md

angela-li committed:
Add link to US SDOH data

angela-li committed:
Merge pull request #32 from GeoDaCenter/us-sdoh

Add link to US SDOH data

angela-li committed:
Remove metadata link for now

Will add documentation with new initdata structure

angela-li committed:
Merge pull request #33 from GeoDaCenter/us-sdoh

Remove metadata link for now

lixun910 committed:
Update index.html

update CHIM.zip download link

lixun910 committed:
Update 2017-07-19-CHIM.md

acostak committed:
Modified description and links of database

acostak committed:
Re-stablish .md name file of Denver and added Abandoned cars data.

acostak committed:
Added Abandoned_cars data and modified some links issues

acostak committed:
Created website-description for us-sdoh data

acostak committed:
Added link of US-SDOH (test)

acostak committed:
Modified us-sdoh link (test)

acostak committed:
changed tags

acostak committed:
Added map to us-sdoh

acostak committed:
hide map us-sdoh (test)

acostak committed:
Added description of commpop and community_pop

acostak committed:
Community pop description

acostak committed:
Correction commpop and community_pop (test)

acostak committed:
modified description commpop and community_pop

acostak committed:
Modify Denver

acostak committed:
changed denver reference and standardized link style (test)

acostak committed:
standardized open link style

acostak committed:
Added geojson to chicago_commpop.zip file

acostak committed:
Add variables description to xxxxx2 datasets

acostak committed:
Add spiral dataset and its description

acostak committed:
modified website mistake south-spirals

acostak committed:
modified date (test)

acostak committed:
sp changed time (test)

acostak committed:
sp modify date and tag (test)

acostak committed:
Updated King County data (old version has incorrect lon-lat) and modified its description.

acostak committed:
Removed Community pop as it only includes one variable and duplicates commpop. Also, rename the latter.

acostak committed:
Added map to US-SDOH

acostak committed:
modified an issue in Airbnb website

acostak committed:
Correction of MSA(2) datasets variables.

acostak committed:
Updated airbnb

acostak committed:
Modifies some datasets and descriptions (accordingly).

acostak committed:
Updated data and variable description. Raw data in geodaData package repository.

acostak committed:
Modified grocery and Hickory1 dataset and their description

acostak committed:
updated main html file

acostak committed:
updated commpop dataset

acostak committed:
updated commpop data set

acostak committed:
Updated datasets and their corresponding description

acostak committed:
Updated datasets description and links

acostak committed:
Updates variables description

acostak committed:
Test-updates names website

acostak committed:
Updated datasets and descriptions

Commit:
Added snow dataset

Commit:
Updated snow0 description

Commit:
Updated individual snow maps and added geojson file to subset pumps dataset

Commit:
updated pumps description

Commit:
updated references

Commit:
Updated references of individual websites

Commit:
test snow4

Commit:
Documentation and script description added

Commit:
test2 snow4

Commit:
test 2 snow4

Commit:
test 3 snow 4

Commit:
modified map symbols snow 1 and 2

Commit:
change description snow master

Commit:
test1 snow2

Commit:
updated links and references

Commit:
Added reference

Commit:
Modified point maps websites

Commit:
Modified Pittsburgh

lixun910 committed:
Update 2020-09-28-snow.md

add tracking to snow

Commit:
Updates pumps description

Commit:
Merge branch 'gh-pages' of https://github.com/GeoDaCenter/data-and-lab into gh-pages

Commit:
Modifies pump description

Commit:
separates reference

Commit:
update snow documentation pdfs

lixun910 committed:
Update index.html

lixun910 committed:
Update index.html

lixun910 committed:
update for indexing

lixun910 committed:
Update tags.html

lixun910 committed:
update relative url

lixun910 committed:
update for tags

lixun910 committed:
Update index.html

lixun910 committed:
Update index.html

lixun910 committed:
Update tags.html

Commit:
Update Grand experiment name

Commit:
update snow documentation

Commit:
update snow documentation

Commit:
update snow documentation

Commit:
Update snow-geodascripts and snow7 files

acostak committed:
update snow

acostak committed:
update Snow

acostak committed:
test table snow

acostak committed:
update snow7 table

acostak committed:
update flma

acostak committed:
Upload Chicago parcels dataset

acostak committed:
Update snow data

acostak committed:
update parcels

acostak committed:
update snow main table

acostak committed:
Update parcels

acostak committed:
update snow

puttingscienceintodatascience committed:
update chi parcel URL and order of datasets

puttingscienceintodatascience committed:
remove duplicate xy fields in table

puttingscienceintodatascience committed:
remove duplicate xy fields in table

puttingscienceintodatascience committed:
Update 2017-07-18-KingCounty HouseSales2015.md

lost --> lots

stuartlynn committed:
Simple docker set up for local dev

Makosak committed:
Merge pull request #36 from stuartlynn/docker

Simple docker set up for local dev

puttingscienceintodatascience committed:
updating Snow data

puttingscienceintodatascience committed:
updated geoJSON for 8 pump rings

puttingscienceintodatascience committed:
updating main Snow data overview page

reflecting changes in pumps and pump rings
new reference

puttingscienceintodatascience committed:
Update pump rings

From 6 to 8 pumps
new variables

puttingscienceintodatascience committed:
updated pumps

from 6 to 8

puttingscienceintodatascience committed:
updated PVJ pest field ref

puttingscienceintodatascience committed:
updating text

puttingscienceintodatascience committed:
updated description

puttingscienceintodatascience committed:
fixed typo

puttingscienceintodatascience committed:
fixed variable name

lixun910 committed:
[Page] add page for Oaxaca dataset (#37)

Signed-off-by: Xun Li <lixun910@gmail.com>

lixun910 committed:
upgrade to GA4 (#38)

dpeachpeach committed:
Upgraded Jekyll version utilized in Dockerfile (#41)

* Upgraded Jekyll version utilized in Dockerfile, this version works in serving locally

* upgrade to gulp v4

---------

Co-authored-by: Xun Li <lixun910@gmail.com>

lixun910 commented:
@dpeachpeach You need to create a new branch based on gh-pages, and then git cherry-pick your commit  [40ee839](https://github.com/GeoDaCenter/data-and-lab/pull/34/commits/40ee8393f96373a50664f4e0d5bcf73fee69fd28) to the new branch. After that, you can create a PR from your new branch.

jkoschinsky committed:
fixed 3 errors on p. 19

lixun910 committed:
Feat: using kepler.gl to replace leaflet map (#42)

lixun910 committed:
Fix: css and logger

Signed-off-by: Xun Li <lixun910@gmail.com>

lixun910 committed:
Site: Add new sample datasets (#46)

lixun910 committed:
Fix: index page and new data pages

jkoschinsky committed:
add buenosaires.geojson so map is displayed

jkoschinsky committed:
Rename 2020-08-13-spirals.md to 2020-08-13-Spirals.md

spirals --> Spirals

lixun910 committed:
[Data] Update dataset for Chi-CCA and Italy Community Bank

lixun910 committed:
[Data] Add US Zipcodes Centroids

lixun910 committed:
[Test] Ai Chatbot

lixun910 committed:
[Fix] css file path contains extra space

lixun910 committed:
[Feat] Ai chatbot on all sample datasets

lixun910 committed:
[Misc] Update Ai insturction

lixun910 committed:
[chore] update openassistant

lixun910 committed:
[Feat] Allow to open/close the AI assistant dialog

lixun910 committed:
[fix] update geojson data of chicago_commpop

lixun910 committed:
[data] add commpopulation.csv direct access

lixun910 committed:
[data] add chicago_sup.csv

lixun910 committed:
[data] add sf_cartheft geojson

lixun910 committed:
[data] add ohlung geojson file

jkoschinsky committed:
Update index.html

removing NTA dataset: wrong aggregation from tracts; users can work with tract data to aggregate to NTA or other areas
